src/component/profileform/index.tsx

```diff
@@ -46,28 +53,20 @@ function ProfileForm(props: ProfileFormProps) {
   });
 
   const values = watch();
-  const { mutate: saveUser } = useCreateUser();
+
+  useEffect(() => {
+    if (onNameChange && values.name) onNameChange(values.name);
+    if (onTitleChange && values.title) onTitleChange(values.title);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
Does this execute on change without a dependency array?

Reply:
it doesn't, but it guarantees the onChange in line 90 and line 109 in this file